web2/src/components/EditDialog.vue

```diff
@@ -81,6 +80,14 @@ export default {
     };
   },
 
+  beforeCreate() {
+    window.addEventListener('keydown', (ev) => {
+      if (ev.key === 'Escape' && this.dialog !== false) {
+        this.close();
+      }
+    });
+  },
+
```

Comment:
I think it is bad idea. This listener must be removed manually when form closed.

Reply:
Ok, I'll try rewrite to mounted/unmounted

Reply:
Rewritten to watcher